### PR TITLE
Change the content of about to match the website 

### PR DESCRIPTION
**Pull request by osaimi**
* change the more card on the top
* add "REQUEST AN EDIT OR REMOVAL"
* add "DID WE MISS SOMEONE?"
* remove "thank you"
* add Arabic strings
* some minor UI refactoring
Simulator Screen Shot - iPhone 11 Pro Max - 2020-06-12 at 18 31 50
Simulator Screen Shot - iPhone 11 Pro Max - 2020-06-12 at 18 31 59
Simulator Screen Shot - iPhone 11 Pro Max - 2020-06-12 at 18 53 56
Simulator Screen Shot - iPhone 11 Pro Max - 2020-06-12 at 18 54 03

### CODE REVIEW

File: SayTheirNames/Source/Controller/More/MoreCard.swift [hunk 1]
```diff
@@ -27,46 +27,49 @@ import UIKit
...
         hashTag.numberOfLines = Theme.Components.LineLimit.single
-        hashTag.translatesAutoresizingMaskIntoConstraints = false
         
-        let vStack = UIStackView(arrangedSubviews: [header, hashTag])
+        let header = UIStackView(arrangedSubviews: [label, hashTag])
+        header.axis = .vertical
+        header.spacing = Theme.Components.Padding.tiny
+        
+        let twitterLogo = UIImageView(image: UIImage(asset: STNAsset.Image.twitterLogo))
+        twitterLogo.heightAnchor.constraint(equalToConstant: 25).isActive = true
```
Comment (Mackis) [inline SayTheirNames/Source/Controller/More/MoreCard.swift:49]: Maybe we can add the `25` to the Theme file.

**Comment by Mackis** [comment]
Looks good to me just the lint warning about line breaks

File: SayTheirNames/Source/Controller/More/MoreCard.swift [hunk 2]
```diff
@@ -27,46 +27,49 @@ import UIKit
...
         
-        NSLayoutConstraint.activate([
-            vStack.centerXAnchor.constraint(equalTo: imageView.centerXAnchor),
-            vStack.centerYAnchor.constraint(equalTo: imageView.centerYAnchor),
-            logo.heightAnchor.constraint(equalToConstant: 30),
-            logo.widthAnchor.constraint(equalToConstant: 20)
-        ])
+        let padding = Theme.Components.Padding.medium
+        
+        vStack.anchor(superView: imageView, top: imageView.topAnchor, leading: imageView.leadingAnchor, bottom: imageView.bottomAnchor, trailing: imageView.trailingAnchor, padding: UIEdgeInsets(padding, padding, padding, padding))
```
Comment (ohwhen) [inline SayTheirNames/Source/Controller/More/MoreCard.swift:72]: Linting warning.

File: SayTheirNames/Source/Controller/More/MoreView.swift [hunk 3]
```diff
@@ -135,17 +164,18 @@ final class MoreView: UIView {
...
+                     bottom: scrollView.bottomAnchor, trailing: scrollView.trailingAnchor,
                            padding: UIEdgeInsets(top: 20, left: 20, bottom: 20, right: 20), size: .zero)
         
         NSLayoutConstraint.activate([
             moreCard.topAnchor.constraint(equalTo: scrollView.topAnchor, constant: 20),
             moreCard.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -20),
             moreCard.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 20),
-            moreCard.heightAnchor.constraint(equalToConstant: 150),
-            thankYouLabel.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor, constant: -Theme.Components.Padding.large)
+            moreCard.heightAnchor.constraint(equalTo: moreCard.widthAnchor, multiplier: 0.6)
```
Comment (ohwhen) [inline SayTheirNames/Source/Controller/More/MoreView.swift:175]: mind reusing the padding from `Theme`?

File: SayTheirNames/Source/Controller/More/RequestOrRemoveTextView.swift [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        showsVerticalScrollIndicator = false
+        showsHorizontalScrollIndicator = false
+        
+        setupAttributedString()
+    }
+    
+    func setupAttributedString() {
+        let text = L10n.RequestEditOrRemoval.details
+        let attributedString = NSMutableAttributedString.createHyperink(for: MoreLinks.twitter, in: text, as: L10n.twitter)
+        attributedString.addAttribute(.font, value: UIFont.STN.ctaBody, range: NSMakeRange(0, attributedString.length))
```
Comment (ohwhen) [inline SayTheirNames/Source/Controller/More/RequestOrRemoveTextView.swift:44]: Linting warning.

File: SayTheirNames/Resources/Generated/strings.swift [hunk 5]
```diff
@@ -13,6 +13,8 @@ import Foundation
 internal enum L10n {
   /// About
   internal static let about = L10n.tr("Localizable", "about")
+  /// Welcome to the Say Their Names Project. Our aim is to build an open-source platform to raise awareness of the injustice and often forgotten names of racial inequality. We strive to identify and amplify verified organizations to ensure donations are reaching those who can make the most impact with it.
```
Comment (Sharkesm) [inline SayTheirNames/Resources/Generated/strings.swift:16]: Can we have a shorter description

File: SayTheirNames/Source/Controller/More/MoreView.swift [hunk 6]
```diff
@@ -212,6 +252,15 @@ private extension MoreView {
 // MARK: - Button Action Responders
 private extension MoreView {
     
+    @objc func openRequestPage(_ sender: Any) {
+        guard let url = URL(string: MoreLinks.contributeToTheList) else {
+            assertionFailure("Failed to create slack url")
```
Comment (Sharkesm) [inline SayTheirNames/Source/Controller/More/MoreView.swift:257]: Update the assertion failure message

File: SayTheirNames/Source/Controller/More/MoreView.swift (hunk 3 above)
Comment (osaimi) [reply SayTheirNames/Source/Controller/More/MoreView.swift:175]: Should I define this multiplier in the theme?

File: SayTheirNames/Resources/Generated/strings.swift (hunk 5 above)
Comment (osaimi) [reply SayTheirNames/Resources/Generated/strings.swift:16]: SwiftGen 🤷‍♂️

**Comment by hybridcattt** [comment]
@osaimi while waiting for another review, could you pull latest translations + add `.localizedUppercase` in code for texts that should be all caps? 🙏  then we can save a roundtrip with another PR 🙂

**Comment by osaimi** [comment]
> @osaimi while waiting for another review, could you pull latest translations + add `.localizedUppercase` in code for texts that should be all caps? 🙏 then we can save a roundtrip with another PR 🙂

Sure.

About the uppercase, can't we just use uppercased()?

**Comment by hybridcattt** [comment]
`uppercased()` doesn't work on non-latin 

> > @osaimi while waiting for another review, could you pull latest translations + add `.localizedUppercase` in code for texts that should be all caps? 🙏 then we can save a roundtrip with another PR 🙂
> 
> Sure.
> 
> About the uppercase, can't we just use uppercased()?

For working with user-entered text, localised version should always be used, because it depends on user's language. [documentation](https://developer.apple.com/documentation/foundation/nsstring/1408467-lowercased) 

For example, the uppercase version of "i" is "I" in English, but "İ" in Turkish. If you always apply English rules, uppercasing of Turkish words will end up wrong.

**Comment by osaimi** [comment]
> `uppercased()` doesn't work on non-latin
> 
> > > @osaimi while waiting for another review, could you pull latest translations + add `.localizedUppercase` in code for texts that should be all caps? 🙏 then we can save a roundtrip with another PR 🙂
> > 
> > 
> > Sure.
> > About the uppercase, can't we just use uppercased()?
> 
> For working with user-presented text in different languages, localised version should always be used, because uppercasing depends on user's language. [documentation](https://developer.apple.com/documentation/foundation/nsstring/1408467-lowercased)
> 
> For example, the uppercase version of "i" is "I" in English, but "İ" in Turkish. If you always apply English rules, uppercasing of Turkish words will end up wrong.

I took care of the conflict. 

I'll look into localizedUppercase. I've never used localizedUppercase before.

**Pull request merged**